cmd/notary/tuf.go

```diff
@@ -546,3 +587,30 @@ func getRemoteTrustServer(config *viper.Viper) string {
 	}
 	return defaultServerURL
 }
+
+func getTrustPinning(config *viper.Viper) (trustpinning.TrustPinConfig, error) {
```

Comment:
I don't see any tests specifically for this method -- there's a decent amount of logic here. Would you mind adding some tests that we properly parse these files and generate the expected `TrustPinConfig`s?